src/target/riscv_debug.c

```diff
@@ -918,6 +927,36 @@ static const char *riscv_fpu_ext_string(const uint32_t extensions)
 	return "";
 }
 
+/*
+ * Generate the fpu section of the description
+ * fpu_size =32 single precision float
+ * fpu_size =64 double precision float
+ */
+static size_t riscv_build_target_fpu_description(size_t fpu_size, char *const buffer, size_t max_length)
+{
+	const size_t first_fpu_register = RV_FPU_GDB_OFFSET;             // fixme!
+	const size_t first_fpu_control_register = RV_FPU_GDB_CSR_OFFSET; // fixme!
```

Comment:
It would be useful with this and the line above if you could expand on what needs fixing about them - it's very curious what those "fixme!" comments are about.

Reply:
No sure i got it correctly, i think the idea was to move the clang-format on up
+ fixed a mistake on fpu csr param that were swapped
Thank you

Reply:
Think you'd meant to reply to the first of these 4 review items. Yes, that is the change we were thinking though you've moved it further up than we were anticipating - but think that's ok as that other code needed some formatting love too.

The only remaining issue there is the lack of a blank line between the preceding array and your addition, but as the CSR stuff needs some work, don't worry about that. We can fix that after.